OpenSim/Simulation/Test/testModelInterface.cpp

```diff
@@ -0,0 +1,133 @@
+/* -------------------------------------------------------------------------- *
+ *                       OpenSim:  testModelInterface.cpp                     *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2017 Stanford University and the Authors                *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#include <OpenSim/Auxiliary/auxiliaryTestFunctions.h>
+#include <OpenSim/Simulation/Model/Model.h>
+#include <OpenSim/Simulation/Model/PhysicalOffsetFrame.h>
+#include <OpenSim/Simulation/Manager/Manager.h>
+#include <OpenSim/Common/LoadOpenSimLibrary.h>
+
+using namespace OpenSim;
+using namespace std;
+
+
+int main() {
+    LoadOpenSimLibrary("osimActuators");
+
+    try {
+        // the specific model used here is not important
+        Model model("arm26.osim");
+        model.finalizeFromProperties();
```

Comment:
This comment was neither addressed nor replied to. Is there another test somewhere else that confirms `!model.isObjectUpToDateWithProperties()` immediately after loading a Model, or was the suggestion ignored because it was deemed unnecessary to perform this check?